vignettes/detection_efficiency_vignette.Rmd

```diff
@@ -0,0 +1,542 @@
+    id:receiver_serial_no,
+    deploy_date_time:expect_deploy_long, geometry
+  )
+```
+
+## Export to excel and gpx formats
+Then we will save as an excel and gpx file. To save as excel we will use [{openxlsx}](https://ycphs.github.io/openxlsx/index.html). You will notice that I don't have it in the load packages area of this vignette. I have not placed it there for the purpose of how R builds vignettes but please add it to your load packages call. You will also need to replace `"YOUR_FILE_PATH"` with your file path for both saving as an excel and/or gpx file. 
+
+```{r, eval = FALSE}
+#| title: save excel and gpx file
+
+# save as excel
+openxlsx::write.xlsx(deploy_sites, "YOUR_FILE_PATH.xlsx")
+
+# save as gpx
+st_write(deploy_sites, "YOUR_FILE_PATH", driver = "GPX")
+```
+
+Now that you've created the excel and gpx file you can head out in the field to deploy your ranges transmitters for a given amount of time (e.g., 24 hr). After the time period you will retriever your receivers, download the vrl files and bring them into fathom central or the older detection range software. Once you export the csv you can move on to the next part of this vignette which is the analysis side.  
+
+## Preliminary data download 
+
+Import vrl files and receiver and transmitter location data into [Fathom Central](https://fathomcentral.com/signin). Unfortunately you will need internet to do this. If you are unable to use the internet (e.g., in the field), download an older version of the detection efficiency software produced by Innovasea and import the vrl and receiver and transmitter location data. The manual for this software is quite good and will walk you through how to get your data into the detection efficiency tool. You can download the older version at the following [Innovasea website](https://support.vemco.com/s/downloads). Alternatively, you can calculate preliminary detection efficiency  in R. 
+
+To do this in R can either read in each vrl file into R using {glatos} or {rvdat} or create a vue database, bring in all vrl files, and export the detection file as a csv. 
+
+Next, add in a column that is the number of detections expect to be heard in 24 hr for the given delays. To do this we can take the number of seconds in a day, 86,400 s, divided by average delay which in this case is 900 s. This value, 96, means that we should hear a 96 times in a day if a receiver heard all detections. 
+
+We can use the following example code to determine the preliminary percentage of detections that were heard at the set distances.  
+
+```{r}
+#| echo: TRUE
+#| eval: FALSE
+
+det_summary <- dets %>%
+  group_by(station, dets_expected, tag_serial_name) %>%
+  summarise(
+    dets_heard = n()
+  ) %>%
+  ungroup() %>%
+  mutate(
+    dets_eff = dets_heard / dets_expected,
+    dets_eff_perc = (dets_heard / dets_expected) * 100
+  )
+```
+
+# Analysis
+
+For the analysis you will want to start a new R script, hence why I have left load packages below. 
```

Comment:
I have amended the vignette to reload and create this object so that it does not error considering the object cannot be found considering it isn't brought in.